Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -128,6 +128,7 @@
   - type: Gun
     fireRate: 2
   - type: BallisticAmmoProvider
+    autoCycle: true
```

Comment:
To be clear, this means the weapon no longer needs the user to click on it to eject a shell, with this set to true? They can just fire one shot after another?